src/Stratis.Bitcoin.Features.BlockStore/BlockStoreFeature.cs

```diff
@@ -82,15 +86,32 @@ private void AddInlineStats(StringBuilder log)
 
             if (highestBlock != null)
             {
-                string logString = $"BlockStore.Height: ".PadRight(LoggingConfiguration.ColumnLength + 1) + highestBlock.Height.ToString().PadRight(8) +
-                             $" BlockStore.Hash: ".PadRight(LoggingConfiguration.ColumnLength - 1) + highestBlock.HashBlock;
-
-                log.AppendLine(logString);
+                if (!this.storeSettings.PruningEnabled)
```

Comment:
why logging is disabled if in pruned node? 
even when we prune we have BSQ tip

Reply:
With the light wallet we never had a block store height but I suppose we can always show seeing as pruning is a FN function. I will fix.